third-party/linshare/src/test/java/org/apache/james/linshare/client/TechnicalAccountCreationRequest.java

```diff
@@ -0,0 +1,73 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.linshare.client;
+
+import static org.apache.james.linshare.LinshareFixture.ACCOUNT_ENABLED;
+import static org.apache.james.linshare.LinshareFixture.TECHNICAL_ACCOUNT;
+
+public class TechnicalAccountCreationRequest {
```

Comment:
Isn't it a DTO?

Reply:
`TechnicalAccountCreationRequest`, `TechnicalAccountGrantPermissionsRequest` are part of 
`TechnicalAccountDTO`

`TechnicalAccountDTO` is a response model from `LinShare`. So i decided to separate them into three classes.

Reply:
> TechnicalAccountCreationRequest, TechnicalAccountGrantPermissionsRequest are part of
TechnicalAccountDTO

What do you mean? It's not what I'm reading in the code

Reply:
Oh sorry, i mean i checked at `Linshare core`. So i will separate it into these classes in James

Reply:
IMO: I preferred to reused the model in Linshare documentation's API so that is why i named class like that.

BTW, I agree with using of `request` and `response` object and i will change it.